README.rst

```diff
@@ -2,27 +2,127 @@
 Parsimonious
 ============
 
-Parsimonious aims to be the fastest arbitrary-lookahead parser written in pure
-Python—and the most usable. It's based on parsing expression grammars (PEGs),
-which means you feed it a simplified sort of EBNF notation. Parsimonious was
-designed to undergird a MediaWiki parser that wouldn't take 5 seconds or a GB
-of RAM to do one page, but it's applicable to all sorts of languages.
+If you've ever learned a programming langauge from formal notation or read the
+docs for a command-line tool, you already know how to use Parsimonious:
```

Comment:
This seems false. At best, you know how to read a parsimonious grammar, but using it requires understanding that it's an PEG implementation and what the particular syntax is.

Honestly, I think the previous introduction is more informative. Why change it?